packages/terra-date-time-picker/src/terra-dev-site/test/date-time-picker/twelve-hour/DateTimePickerDisabled12HourMobile.test.jsx

```diff
@@ -36,15 +32,15 @@ export default class DateTimePicker12HourDisabledMobile extends React.Component
 
   render() {
     return (
-      <div className={cx('content-wrapper')}>
+      <>
```

Comment:
why was this `div` dropped? We don't normaally wrap with `<>`

Reply:
`<>` is a shorthand for `<React.Fragment>`. Having some wrapper is needed since the fragment has two children but it doesn't need to be a div because the `div` was needed to apply the `className`.